demo/DemoApp.js

```diff
@@ -86,6 +86,10 @@ class DemoApp extends React.Component {
     }
   }
 
+  getRowKey({ rowIndex }) {
+    return 'row_' + rowIndex + '_key_' + Math.floor(Math.random() * 10000000);
```

Comment:
let's make the key string `row_key_${this.state.data[rowIndex].id}` to provide a better example. usually prefer not to use random and index in the key.